composer.json

```diff
@@ -42,7 +51,7 @@
 		"npm-asset/cropperjs": "1.2.2",
 		"npm-asset/imagesloaded": "4.1.4",
 		"monolog/monolog": "^1.24"
-	},
+    },
```

Comment:
Shouldn't this be a tab instead of 4 spaces?

Reply:
Yup, good catch.